aiida/cmdline/commands/exportfile.py

```diff
@@ -41,20 +41,33 @@ def cli(self, *args):
     help='Export the given groups by pk')
 @click.option('-g', '--group_names', multiple=True, type=str,
     help='Export the given groups by group name')
-@click.option('-P', '--no-parents', is_flag=True, default=False,
-    help='Store only the nodes that are explicitly given, without exporting the parents')
-@click.option('-O', '--no-calc-outputs', is_flag=True, default=False,
-    help='If a calculation is included in the list of nodes to export, do not export its outputs')
+@click.option('-IF', '--input_forward', is_flag=True, default=False,
+    help='Follow forward INPUT links (recursively) when calculating the node '
+         "set to export. By default this is switched off.")
+@click.option('-CrR', '--create_reversed', is_flag=True, default=True,
```

Comment:
You can use `show_default=True` to have `click` automatically generate the default value in the help string